tests/Support/TestHelperTrait.php

```diff
@@ -0,0 +1,62 @@
+<?php
+namespace Tests\Support;
+
+use App\Record;
+use App\RecordIssuer;
+use App\RecordIssuerType;
+use App\User;
+use Illuminate\Contracts\Console\Kernel;
+use Mockery\CountValidator\Exception;
+
+trait TestHelperTrait
+{
+    // return s array of users
+    private function generateUsersInDatabase($count = 1) {
+        return factory(User::class, $count)->create();
+    }
+
+    private function getBillOrgTypeId() {
+        return RecordIssuerType::type(RecordIssuerType::BILLORG_TYPE_NAME)->first()->id;
+    }
+
+    private function createRecordIssuerModel($organizationName) {
+        $recordIssuer = new RecordIssuer();
+        $recordIssuer->name = $organizationName;
+        $recordIssuer->type = $this->getBillOrgTypeId();
+        return $recordIssuer;
+    }
+
+    // precondition: user must exist
+    private function generateOrganizationWithFactoryForUser($orgTypeName, User $user){
+        $org = $this->generateOrganizationsWithFactoryForUser($orgTypeName, $user, 1)[0];
+        return $org;
+    }
+
+    private function generateBillOrgs($user, $count){
+        $orgTypeName = RecordIssuerType::BILLORG_TYPE_NAME;
+        return $this->generateOrganizationsWithFactoryForUser($orgTypeName, $user, $count);
+    }
+
+    private function generateBanks($user, $count) {
+        $orgTypeName = RecordIssuerType::BANK_STATEMENT_TYPE_NAME;
+        return $this->generateOrganizationsWithFactoryForUser($orgTypeName,$user,$count);
+    }
+
+    private function generateOrganizationsWithFactoryForUser($orgTypeName, User $user, $count){
```

Comment:
Correct me if i'm wrong, but the `$orgs` returned won't have any ID in it, right? Meanwhile we may need the IDs to link record to record issuer.